<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtFFmpegPlayerClass</class>
 <widget class="QMainWindow" name="QtFFmpegPlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtFFmpegPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="XVideoWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>9</y>
      <width>961</width>
      <height>543</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openFile">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>500</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="isPlay">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>500</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="XSlider" name="playPos">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>711</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XVideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>XVideoWidget.h</header>
  </customwidget>
  <customwidget>
   <class>XSlider</class>
   <extends>QSlider</extends>
   <header>XSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QtFFmpegPlayer.qrc"/>
 </resources>
 <connections/>
</ui>
